package.json

```diff
@@ -120,11 +120,12 @@
     "react-test-renderer": "^16.8.6",
     "regenerator-runtime": "^0.12.1",
     "rimraf": "^2.6.2",
-    "rollup": "^0.66.6",
+    "rollup": "^2.16.1",
```

Comment:
We’ve held off on upgrading rollup because there’s a bunch of unsupported peerDependencies—did you see anything strange on your end?

Reply:
Unsupported peer dependencies of what exactly?  The plugin I added just requires that a user has `React` and `ReactDom` installed alongside radiance-ui.  e.g. we're not going to bundle it within radiance. (For the ESMs  anyways)

Reply:
For example `rollup-plugin-node-resolve ` now exists under the `@rollup` namespace, I think v2 moved most of the supporting modules there.

Reply:
I see.  I'll take a look and see if there's anything else that needs updating.